dxr/static/js/dxr.js

```diff
@@ -428,6 +428,22 @@ $(function() {
     }
 
 
+    // Show the help box when the help icon is clicked, and hide it when
+    // anything outside of the box is clicked.
+    var helpIcon = $('.help-icon'),
+        helpMsg = helpIcon.find('.help-msg');
+
+    helpIcon.on('click', function() {
+        helpMsg.fadeIn(300);
```

Comment:
Let's not fade it in; we don't do that anywhere else (for the perception of speed).